version1/02 - Object types.md

```diff
@@ -609,6 +615,102 @@ The following notation is used in the tables below:
 |           | ref       | refDokument            |    x     |             |               |  |      |  x   |   x   |      |  x   |   x    |                                                                    |
 |           | ref       | refEksternId           |          |             |               |  |      |      |   x   |      |  x   |        | &nbsp;                                                             |
 
+## National Identifiers
```

Comment:
Each type can also have the `beskrivelse` (description) property.

Reply:
Addressed with [this commit](https://github.com/documaster/noark5-web-services/pull/37/commits/01b71b03b04b6a811a1355472d58ff30d84e2da7).